cfgov/jinja2/v1/_includes/molecules/social-media.html

```diff
@@ -0,0 +1,137 @@
+{# ==========================================================================
+
+   social_media.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates markup for Social Media molecule.
+
+   is_share_view: Whether the "Share this" heading is shown.
+                  Default is true.
+
+   options:       Object of optional parameters.
+
+   options.title:            Sets the title in email subject line
+                             and in the LinkedIn share post.
+                             Default is "Look what I found on the CFPB!"
+
+   options.twitter_hashtags: A comma-separated list of hashtags
+                             to be appended to default Tweet text.
+                             https://dev.twitter.com/web/tweet-button/parameters
+
+   options.twitter_related: A comma-separated list of accounts
+                            related to the content of the shared URI.
+                            https://dev.twitter.com/web/tweet-button/parameters
+
+   options.twitter_lang:    Loads text components in the specified language.
+                            https://dev.twitter.com/web/tweet-button/parameters
+
+   ========================================================================== #}
+
+{% macro render(is_share_view=true, options={}) %}
+{% set title  =  options.title | default('Look what I found on the CFPB!') %}
+<div class="m-social-media">
+    {% if is_share_view %}
+        <h5 class="m-social-media_heading">Share this</h5>
```

Comment:
Updated to `<p class="h5` in https://github.com/cfpb/cfgov-refresh/commit/3fdaca66cc9a903f575d192d489a55938375924e